src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Ball(int number, String color) {
+        this.number = number;
+        this.color = color;
+    }

```

Comment:
It is unnecessary to create two constructors with the same params but with a different ordering. Let's keep only one.